src/pages/SeriesDetailsPage.js

```diff
@@ -85,6 +104,12 @@ export default function SeriesDetailsPage({
       )}
     </Wrapper>
   );
+
+  function handleNavigation() {
+    currentPage === PAGES.DETAILS
+      ? setCurrentPage(PAGES.SEASONS)
+      : setCurrentPage(PAGES.DETAILS);
+  }
```

Comment:
And here add the `page` to your state :)
```suggestion
  function handleNavigation(page) {
    setCurrentPage(page)
  }
```

Reply:
Awesome! Thank you very much! 🚀